scripts/translation-gpt/translate.mjs

```diff
@@ -1,4 +1,5 @@
 import {existsSync, openSync, readFileSync, writeFileSync} from 'fs';
+import {EOL} from 'os';
```

Comment:
Mh watch out, this is OS bound, meaning that the generated text would change depending on the runner OS.
I would propose to append `\n` instead

Reply:
Yes I thought making it OS bound might be a good thing? We do use it when generating commit msg e.g.
Guess we want to use EOL in cmd line utils etc but not in file content?